electron_app/src/webcontents-handler.js

```diff
@@ -84,14 +84,6 @@ function _CutCopyPasteSelectContextMenus(params) {
     }];
 }
 
-function onSelectedContextMenu(ev, params) {
```

Comment:
This is what you overrode too though: (Select, vs Input)
![image](https://user-images.githubusercontent.com/2403652/26967828-9c9b6314-4cf7-11e7-9c7f-a892158cac3b.png)